build.gradle

```diff
@@ -472,37 +472,20 @@ task copyTestResources(type: Copy) {
 processTestResources.dependsOn copyTestResources
 
 tasks.withType(Test) {
-    reports.html.destination = file("${reporting.baseDir}/${name}")
+    reports.html.outputLocation.set(file("${reporting.baseDir}/${name}"))
 }
 
-task jacocoMergePrep() {
-    doFirst {
-        // Ignore failures of tests
-        tasks.withType(Test) {
```

Comment:
Without this the codecoverage workflow is now failing due to failing tests.

Reply:
I removed it because jacocoMerge is deprecated. We probably have to think of another way to restore the codecoverage workflow.

Reply:
Codecoverage should work again now...